### PR TITLE
Feature 166/context refactor

### PR DESCRIPTION
**Pull request by chrismckiernan**

### CODE REVIEW

File: server/src/main/java/com/objectcomputing/checkins/services/memberprofile/MemberProfileController.java [hunk 1]
```diff
@@ -76,11 +76,13 @@ public MemberProfileController(MemberProfileServices memberProfileServices){
      * @param name
```
Comment (Bagurp) [inline server/src/main/java/com/objectcomputing/checkins/services/memberprofile/MemberProfileController.java:76]: we probably want to include workEmail on line 75

File: server/src/test/java/com/objectcomputing/checkins/services/memberprofile/MemberProfileControllerTest.java [hunk 2]
```diff
@@ -33,7 +30,7 @@
 public class MemberProfileControllerTest {
```
Comment (Bagurp) [inline server/src/test/java/com/objectcomputing/checkins/services/memberprofile/MemberProfileControllerTest.java:30]: Do we need tests for findByWorkEmail?

File: server/src/main/java/com/objectcomputing/checkins/services/memberprofile/MemberProfileController.java (hunk 1 above)
Comment (chrismckiernan) [reply server/src/main/java/com/objectcomputing/checkins/services/memberprofile/MemberProfileController.java:76]: good catch done

File: server/src/test/java/com/objectcomputing/checkins/services/memberprofile/MemberProfileControllerTest.java (hunk 2 above)
Comment (chrismckiernan) [reply server/src/test/java/com/objectcomputing/checkins/services/memberprofile/MemberProfileControllerTest.java:30]: good catch added

File: web-ui/src/api/api.js [hunk 3]
```diff
@@ -1,16 +1,18 @@
...
-    try {
-      resolved.payload = await promise;
-    } catch(e) {
-      resolved.error = e;
-    }
-  
-    return resolved;
-  }
\ No newline at end of file
+export const resolve = async (promise) => {
```
[the remark this answers is not in the document]
Comment (chrismckiernan) [reply web-ui/src/api/api.js:3]: Added.

File: web-ui/src/components/personnel/Personnel.jsx [hunk 4]
```diff
@@ -7,100 +7,125 @@ import "./Personnel.css";
...
+      infoClassName = "personnel-info";
     }
 
     return (
       <div key={key} className="image-div">
-        <img className="member-image" alt="personnel pic" src={require("../../images/default_profile.jpg")}/>
+        <img
+          className="member-image"
+          alt="personnel pic"
+          src={require("../../images/default_profile.jpg")}
```
[the remark this answers is not in the document]
Comment (chrismckiernan) [reply web-ui/src/components/personnel/Personnel.jsx]: Done.

**Pull request merged**